pom.xml

```diff
@@ -20,20 +20,40 @@
   mvn versions:update-properties
   -->
   <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <!-- FDO Dependency Version -->
+    <fdo.version>1.0-SNAPSHOT</fdo.version>
+
+    <!-- Dependency Version -->
+    <bcpkix.version>1.69</bcpkix.version>
+    <bcprov.version>1.69</bcprov.version>
+    <commons-beanutils.version>1.9.4</commons-beanutils.version>
+    <commons-codec.version>1.15</commons-codec.version>
+    <commons-configuration2.version>2.7</commons-configuration2.version>
+    <dbcp2.version>2.8.0</dbcp2.version>
+    <exec-maven-plugin.version>3.0.0</exec-maven-plugin.version>
+    <h2.version>1.4.200</h2.version>
+    <javax.servlet-api.version>4.0.1</javax.servlet-api.version>
     <junit-jupiter.version>5.7.1</junit-jupiter.version>
-    <slf4j-api.version>1.7.30</slf4j-api.version>
+    <junit-jupiter.version>5.8.0-M1</junit-jupiter.version>
+    <log4j.version>2.14.1</log4j.version>
+    <tomcat.version>9.0.46</tomcat.version>
+
```

Comment:
`jackson`  and `slf4j` dependencies are not included in the root pom.xml. Is it intentional?